ci/workflows/pull-request.yml

```diff
@@ -19,7 +19,7 @@ steps:
   - label: "build"
     command: "ci/scripts/build.sh -p ci-dev"
     key: "build"
-    if: (!build.pull_request.labels includes "ci/skip-ci" || build.pull_request.labels includes "ci/run-build")
+    if: (!(build.pull_request.labels includes "ci/skip-ci" || build.env("CI_STEPS") != null) || build.pull_request.labels includes "ci/run-build" || build.env("CI_STEPS") =~ /,build,/)
```

Comment:
```suggestion
    if: (!(build.pull_request.labels includes "ci/skip-ci" || build.env("CI_STEPS") != null) || build.pull_request.labels includes "ci/run-build" || build.env("CI_STEPS") =~ /build,/)
```

Reply:
Can we just use a trailing comma?

Reply:
If we can assume one name would never be the suffix of another name, then yes. The trailing comma is here to avoid matching and running `/build/` when input is `build-other`. I simply included the leading trailing comma without checking if such cases do exist.

Reply:
Just went through and the assumption does not hold: `check` is a suffix of `misc-check`